ballista/rust/scheduler/src/state/mod.rs

```diff
@@ -242,6 +242,18 @@ impl SchedulerState {
             .collect()
     }
 
+    pub async fn get_tasks_with_job_id(
```

Comment:
does this need to be a public function? looks like it's only used in internal code